addons/popochiu/popochiu_plugin.gd

```diff
@@ -106,13 +106,16 @@ func _enter_tree() -> void:
 	
 	scene_changed.connect(main_dock.scene_changed)
 	scene_closed.connect(main_dock.scene_closed)
-	# ================================================== Connect to signals ====
+	# ====================================================================== Connect to signals ====
 	
 	if _editor_interface.get_edited_scene_root():
 		main_dock.scene_changed(_editor_interface.get_edited_scene_root())
 	
 	main_dock.setup_dialog.es = _editor_interface.get_editor_settings()
+	
+	# Connect signals between other nodes
 	main_dock.setup_dialog.gui_selected.connect(_gui_templates_helper.copy_gui_template)
+	_gui_templates_helper
```

Comment:
Silly question: I can't get this like syntax... what are you doing here? :no_mouth:

Reply:
Hahaha. The only silly thing here is me. I forgot to delete that line.